### PR TITLE
fix: report right command execution status

### PR DESCRIPTION
**Pull request by CristiCanizales**
### What does this PR do?
- Reports the command execution the right way

### Functionality Before
- steps would always show failure notification

### Functionality After
- steps show right notification

[skip-validate-pr]

### CODE REVIEW

File: packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts [hunk 1]
```diff
@@ -444,9 +443,9 @@ export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
   ) {
     channelService.streamCommandOutput(execution);
     channelService.showChannelOutput();
-    notificationService.reportExecutionError(
```
Comment (diyer) [inline packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts:447]: can you explain this change please :)

Comment (CristiCanizales) [reply packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts:447]: so the function we were calling before was 'reportExecutionError', which as the name says is used for reporting errors. I just changed it to the other function that is used to report general command execution

**Review by daphne-sfdc — APPROVED** [review]
Approved! 🎉

I ran `SFDX: Create and Set Up Project for ISV Debugging` and was able to go through all the steps without seeing any error notifications.  The project was created successfully.

**Review by peternhale — APPROVED** [review]

File: packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts (hunk 1 above)
Comment (diyer) [reply packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts:447]: got it. so now we will get a notification on command execution status?

**Pull request merged**